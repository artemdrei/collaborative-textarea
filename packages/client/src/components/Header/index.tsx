import React from 'react';

import labels from '@root/i18n';

import s from './styles.scss';

const Header = () => (
  <header className={s.header}>
    <div className={s.container}>
      <h1 className={s.title}>{labels.pages.editor.title}</h1>
    </div>
  </header>
);

export default Header;
