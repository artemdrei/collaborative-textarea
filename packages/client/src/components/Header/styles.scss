.header {
  display: flex;
  align-items: center;
  justify-content: flex-start;
  height: 60px;
  padding: 4px 20px;
  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23);
}

.title {
  font-size: 20px;
  margin: 0;
}
