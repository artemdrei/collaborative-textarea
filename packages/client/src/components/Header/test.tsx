import React from 'react';
import { render } from '@testing-library/react';

import Header from './index';

describe('Header', () => {
  it('should take a snapshot', () => {
    const { getByText } = render(<Header />);

    expect(getByText('Form for collaboration')).toMatchSnapshot();
  });
});
