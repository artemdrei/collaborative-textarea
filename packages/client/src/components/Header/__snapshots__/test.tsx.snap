// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should take a snapshot 1`] = `
<h1>
  Form for collaboration
</h1>
`;
